asmcomp/cmm.mli

```diff
@@ -134,10 +134,10 @@ and expression =
   | Clet of Ident.t * expression * expression
   | Cassign of Ident.t * expression
   | Ctuple of expression list
-  | Cop of operation * expression list
+  | Cop of operation * expression list * Debuginfo.t
   | Csequence of expression * expression
   | Cifthenelse of expression * expression * expression
-  | Cswitch of expression * int array * expression array
+  | Cswitch of expression * int array * expression array * Debuginfo.t
```

Comment:
(The point being that if it's Cifthenelse on a variable, then there is a comparison operator in the first subexpression containing debug info; however for Cswitch, that is absent.)